## Scientific Notation and Large Numbers

Below we first examine the results of the computations on the calculator for problems 5 - 7:

5. $(6.023 \times 10^{23}) * (14 \times 10^{-6})$


6. $\frac{48.967 \times 10^6}{4.75 \times 10^{-3}}$


7. $\frac{4.271 \times 10^{-11}}{5.39 \times 10^{12}}$

In [1]:
6.023 * 10**(23)*14*10**(-6)

8.432199999999999e+18

In [4]:
(48.967*10**6)/(4.75 * 10**(-3))

10308842105.263159

In [5]:
(4.271 * 10**(-11))/(5.39 * 10**(12))

7.923933209647494e-24

### Checking Our Answers

We can check to see if how we interpret our answers are correct by actually multiplying the numbers out.  Suppose my answers are:

1. $8.43219... \times 10^{18}$


2. $1.0308... \times 10^{10}$


3. $7.9239... \times 10^{-24}$


I can multiply these on the calculator and compare what I get to the above answers.


In [9]:
8.432199999999999 * 10**(18)

8.432199999999998e+18

In [10]:
1.0308842105263159*10**(10)

10308842105.263159

In [11]:
7.923933209647494*10**(-24)

7.923933209647494e-24

Everything seems to have worked out.  Now we will look at how to check by hand.  First, remember our discussion about exponents.  We are going to use the fact that for any like base, we multiply these numbers by adding the exponents, i.e.:

$$a^b * a^c = a^{b+c}$$

For example, we have two exponents with like bases in \#5 with $10^{23}$ and $10^{-6}$.  To multiply these two numbers, we add the exponents $23 + (-6) = 17$ and the result is $10^{17}$.

From start to finish, we could evaluate this problem by hand as follows.  We apply the laws of multiplication about being able to reorder numbers, break this into two parts; a multiplication problem that we solve with our rules of exponents and a familiar decimal multiplication problem as follows:

* Rearrange the terms based on the fact that order doesn't matter when we're multiplying many numbers: 

$$(6.023 \times 10^{23}) * (14 \times 10^{-6}) = (6.023 \times 14)*(10^{23} \times 10^{-6})$$


* Apply our laws of exponents to the base ten terms: 

$$10^{23} \times 10^{-6} = 10^{17}$$

* Multiply the other pair of numbers:

$$(6.023 \times 14) = 84.32$$

* Put things back together:

$$84.322 \times 10^{17}$$

* Write in scientific notation (remember our leading number should be between 1 and 10):

$$8.4322 \times 10^{18}$$

---

The alteration with the division problems is that we will break the situation into two division problems, and apply the law of exponents for division:

$$ \frac{a^b}{a^c} = a^{b-c}$$